crates/fluvio-hub-util/src/cmd/list.rs

```diff
@@ -78,7 +91,7 @@ mod output {
     impl TableOutputHandler for ListConnectors {
         /// table header implementation
         fn header(&self) -> Row {
-            Row::from(["CONNECTOR", "Visibility"])
+            Row::from(["Connector", "Visibility", "Released"])
```

Comment:
`Published`

Reply:
Probably `Connector` -> `Versions`

Reply:
Updated the Published already! About `Versions` I think that would belong to a new column doesn't it? Perhaps as an iteration from this we could split name from version and use a column for versions only?